tests/integration-tests/helix-payloads/cli-integration-tests.sh

```diff
@@ -2,47 +2,25 @@
 
 set -e
 
-if [ $# -ne 2 ] || [ -z "$1" ] || [ -z "$2" ]; then
+if [ "$#" -ne 2 ] || [ -z "$1" ] || [ -z "$2" ]; then
     echo "The script expects 2 arguments: where to store result logs and where to upload test results (ie. \$HELIX_WORKITEM_UPLOAD_ROOT \$HELIX_WORKITEM_ROOT)" 1>2
     exit 1
 fi
 
-version='1.0.0-ci'
-
-# Clean the NuGet cache from the previous 1.0.0-ci version of the tool
-# TODO: This might have a better solution: https://github.com/dotnet/xharness/issues/123
-echo "Cleaning the NuGet cache from the previous version of the tool..."
-
-# Call dotnet to get rid of the welcome message since the nuget command doesn't respect the --no-logo
-dotnet nuget locals All -l
-cache_dirs=`dotnet nuget locals All -l | cut -d':' -f 2 | tr -d ' '`
-while IFS= read -r path; do
-    echo "Purging cache in $path..."
-    rm -vrf "$path/microsoft.dotnet.xharness.cli/$version"
-    rm -vrf "$path/Microsoft.DotNet.XHarness.CLI/$version"
-done <<< "$cache_dirs"
-
 set -x
 
 here="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+export DOTNET_ROOT=$(dirname $(which dotnet))
+dotnet tool install --no-cache --tool-path "$here/xharness-cli" --version "1.0.0-ci" --add-source "$here" Microsoft.DotNet.XHarness.CLI
```

Comment:
`$here` = HELIX_WORKITEM_ROOT.

While this will work, for real usage I'd recommend installing it to $HELIX_CORRELATION_PAYLOAD/xharness-cli so that it's cached for multiple work items in the same job.

Reply:
Good to know for future reference but here I think it's fine since we only have 1 work item per job always. I would have to pass the variable in as I am not sourcing and then check for existence of the folder so I will just leave it as is since it will get replaced with Helix SDK anyway